.github/workflows/task-get-linux-configurations.yml

```diff
@@ -107,6 +109,16 @@ jobs:
               'OS': 'amazonlinux:2',
               'pre-deps': "yum install -y tar gzip"})
 
+          # amazonlinux:2023 needs pre-checkout dependencies
+          if x86_platforms.count('amazonlinux:2023') > 0:
```

Comment:
Why `count` and not `in`?

Reply:
```suggestion
          if 'amazonlinux:2023' in x86_platforms:
```

Reply:
I think it was copilot suggestion… let’s fix it on all the cases so they are aligned

Reply:
Done.